contracts/pool/PoolFactory.sol

```diff
@@ -78,22 +86,15 @@ contract PoolFactory is IPoolFactory, ReentrancyGuard, PoolErrors {
             _tokenSymbol
         );
 
-        if (blendedPoolAddress == address(0)) revert BlendedPoolNotCreated();
         blendedPool = blendedPoolAddress;
 
         emit BlendedPoolCreated(_asset, blendedPoolAddress, msg.sender);
     }
 
```

Comment:
Consider adding getter for all pools, that would be really good feature @ no cost
```suggestion

    /// @notice Return addresses of all pools 
    function getPools() external override view returns (address[] memory) {
        return poolSet. values();
    }
    
```